ppr-ui/src/utils/mhr-api-helper.ts

```diff
@@ -417,3 +417,75 @@ export async function submitMhrTransfer (payloadData, mhrNumber) {
     }
   }
 }
+
+export async function fetchMhRegistration (
+  mhRegistrationNum: string
+): Promise<any> {
+  const url = sessionStorage.getItem('MHR_API_URL')
+  const config = { baseURL: url, headers: { Accept: 'application/json' } }
+
+  return axios
+    .get(`registrations/${mhRegistrationNum}`, config)
```

Comment:
I think you want the current state of an MH registration: append a request parameter "?current=true". Otherwise you will get the information for the original MH registration.